README.md

```diff
@@ -138,6 +138,7 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
 
 | Extension | Author |
 | :-- | :-- |
+| <img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/angular.svg#gh-light-mode-only" alt="Angular" align=left width=24 height=24><img src="https://raw.githubusercontent.com/simple-icons/simple-icons/develop/assets/readme/angular.svg#gh-dark-mode-only" alt="Angular" align=left width=24 height=24> [Angular Package](https://github.com/avmaisak/ngx-simple-icons) | [@avmaisak](https://github.com/avmaisak) |
```

Comment:
I might have been missing Angular terminology here. Should it be package or module?

Reply:
package  - it's a distribution method

Reply:
You can rename it then :+1:

Reply:
> You can rename it then 👍

fixed